fn_ansible/setup.py

```diff
@@ -0,0 +1,33 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup, find_packages
+
+setup(
+    name='fn_ansible',
+    version='1.0.0',
+    license='MIT',
+    author='IBM Resilient Labs',
+    author_email='resil.labs@gmail.com',
+    url='https://github.com/ibmresilient/resilient-community-apps',
+    description="Simple agentless IT automation tool.",
+    long_description="Automation scripts are written in YAML files called playbook, which are besically tasks and instructions. Those tasks can be performed on a remote system from resilient using this function.",
+    install_requires=[
+        'resilient_circuits>=30.0.0'
```

Comment:
* Need to add yaml and ansible here with the minimum versions this integration was developed with
* Ansible is currently v2.7.9 on PyPi: https://pypi.org/project/ansible/
* Yaml: what package did you use? Is it on PyPi?